packages/components/organisms/f-checkout/src/components/Checkout.vue

```diff
@@ -445,6 +436,10 @@ export default {
 
         formattedMobileNumberForScreenReader () {
             return this.customer.mobileNumber ? [...this.customer.mobileNumber].join(' ') : '';
+        },
+
+        restaurantMenuPageUrl () {
+            return `restaurants-${this.restaurant.seoName}/menu`;
         }
```

Comment:
Is there any specific benefit to having the `restaurantMenuPageUrl` here and injecting it via a `prop`? `Error.vue` could be the one doing this, since the information needed is in the Vuex store.

Reply:
Ill have a check as this was already written before i took over the pr.

Reply:
@SachaZvetelman @kevinrodrigues I moved it from ErrorDialog.vue as now it is in use in two places (Error.vue and ErrorDialog.vue)

Reply:
Thanks!